event-app-api/server.js

```diff
@@ -3,10 +3,9 @@ import cors from "cors";
 import { sequelize } from "./database.js";
```

Comment:
In this file I still see the use of res.json and res.send to send errors. Do you have a specific reason to use one over the other in each circumstance? Otherwise they should be consistent.